docs/notebooks/cpu_vs_gpu.ipynb

```diff
@@ -0,0 +1,297 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "31028ca2-8b9f-43bd-8204-1890f2b0b210",
+   "metadata": {},
+   "source": [
+    "# CPU vs GPU backend\n",
+    "\n",
+    "In this notebook, we compare the use of CPU versus GPU backend in terms of computational efficiency. Note that to be able to run the notebook with GPU backend, you will need an NVIDIA GPU and having followed the installation instructions specific to GPU."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "d7888211-1688-40ac-9b64-505f3b618553",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib notebook"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "d9f7f056-6514-45dc-94ae-7de2ebe55d13",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DEVICE = \"cpu\"\n",
+    "\n",
+    "if DEVICE == \"cpu\":\n",
+    "    BACKEND = \"numpy\"\n",
+    "elif DEVICE == \"gpu\":\n",
+    "    BACKEND = \"cupy\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "d66c45cb-6451-4f56-9f1a-3a723780f7af",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from debiased_spatial_whittle.backend import BackendManager\n",
+    "\n",
+    "BackendManager.set_backend(BACKEND)\n",
+    "\n",
+    "np = BackendManager.get_backend()\n",
```

Comment:
`np` is easily confused with numpy. Convention is to use `xp` for either `numpy` or `cupy`.

```suggestion
    "xp = BackendManager.get_backend()\n",
```

Reply:
Thank you for the suggestion, this might indeed be confusing to some users. I should make the change throughout the docs examples though. I will look into this.